glances/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  glances:
+    image: ${GLANCES_IMAGE}
+    restart: unless-stopped
+    pid: host
+    privileged: true
```

Comment:
We should not use privileged: true, instead follow SECURITY.md and find the minimum capabilities required.

Reply:
https://github.com/EnigmaCurry/d.rymcg.tech/blob/master/SECURITY.md#drop-system-capabilities

Reply:
Fixed in [e86bce8](https://github.com/EnigmaCurry/d.rymcg.tech/pull/284/commits/e86bce8c4914bb96cc0a5d4f9d7427fc8c3c1ac5)